### DISCUSSION TITLE
trying to load a mac os Ventura or Monterey installer on a portable usb drive for my MacBook pro mid 2012

### DISCUSSION THREAD

**Discussion by Chase87** [community/community#43991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i'm able to do everything as instructed in videos but every time i try to creat an installer with two different drives and one older external hard drive.  i get an error at different points some make it to 308mb some to 680mb and sometimes to 1300-1600 all varies but always errors.  am i doing something wrong? i have erased all drives tried formatting as guid and master boot but other changes.

**Comment by Baggycat-is-mine** [comment]
Why Masterboot Record? You need to format as HFS+ Journaled.
If you put Monterey into the Applications folder on the 2012 running Catalina with Open Core on that as well, you simply create the installer from there. Create the installer on the drive you want to use, then it will ask you to build and install open core. 
You have to do this on the Mac you want to install onto, you can't do this for a different mac as each one is created differently. Watch out for the pop up helper wanting permission, that will appear at 600mb. If you leave it unattended it can quit if it doesn't get a response.

**Comment by Baggycat-is-mine** [comment]
I'll link this here:
One problem that I get every time is that if I install Monterey and then try to use Migration Assistant, it gives up a Setup User and asks for a password... that's seemingly impossible to answer and therefore MA fails.
The only way I've been able to avoid this with a new drive is to do an install of the original maximum native OS, i.e. High Sierra or Catalina, migrate from the old drive or a backup, and then upgrade to Monterey with Open Core... it's clunky, time consuming but it works. If there's a solution to this problem I'd love to hear it.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Chase87 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
